qsiprep/interfaces/fmap.py

```diff
@@ -1179,8 +1182,7 @@ def _run_interface(self, runtime):
         LOGGER.info('Generating a PEPOLAR visual report')
 
         # Get a segmentation from an undistorted image as a reference
-        ref_segmentation = self.inputs.t1w_seg if not \
-            isdefined(self.inputs.t2w_seg) else self.inputs.t2w_seg
+        ref_segmentation = self.inputs.t1
```

Comment:
Might have done this after removing FAST and before figuring out the FSL_DIR environment variable trick. Will add back and determine if I need to change anything else to have the segs (either add FAST back in for now or add atropos)

Reply:
These segmentations are used to generate the contour lines in the distortion correction animation. It has to be a segmentation otherwise the lines won't make sense

Reply:
Gotcha, I'll add atropos for T1 following the same pattern as with the T2